Libraries/pbxbuild/CMakeLists.txt

```diff
@@ -13,6 +13,7 @@ add_library(pbxbuild SHARED
             Sources/DerivedDataHash.cpp
```

Comment:
> Auxiliary files were attached to invocations but had no relation to
the containing invocation.

I see, so auxiliary files are basically used by Ninja, not the tools that Ninja invokes.

I actually thought it was nice that invocations contained their auxiliary files, since in a way such files a dependency for the invocation.

Oh well.  LGTM